owncloudData/src/main/java/com/owncloud/android/data/files/datasources/implementation/OCLocalFileDataSource.kt

```diff
@@ -134,12 +141,16 @@ class OCLocalFileDataSource(
     override fun renameFile(fileToRename: OCFile, finalRemotePath: String, finalStoragePath: String) {
         fileDao.moveFile(
             sourceFile = fileToRename.toEntity(),
-            targetFile = fileDao.getFileById(fileToRename.parentId!!)!!,
+            targetFolder = fileDao.getFileById(fileToRename.parentId!!)!!,
```

Comment:
*The shouting plugin when you write a `!!` didn't like this line 💔

Reply:
Not my best line of code, that's for sure 😢